plugins/octopus-deploy/src/components/EntityPageOctopusDeploy/EntityPageOctopusDeploy.tsx

```diff
@@ -15,18 +15,23 @@
  */
 import { useEntity } from '@backstage/plugin-catalog-react';
 import { useReleases } from '../../hooks/useReleases';
-import { getAnnotationFromEntity } from '../../utils/getAnnotationFromEntity';
+import {
+  getProjectIdAnnotationFromEntity,
+  getSpaceIdAnnotationFromEntity,
+} from '../../utils/getAnnotationFromEntity';
 import React from 'react';
 import { ReleaseTable } from '../ReleaseTable';
 
 export const EntityPageOctopusDeploy = (props: { defaultLimit?: number }) => {
   const { entity } = useEntity();
 
-  const projectId = getAnnotationFromEntity(entity);
+  const projectId = getProjectIdAnnotationFromEntity(entity);
+  const spaceId = getSpaceIdAnnotationFromEntity(entity);
```

Comment:
yeah as mentioned in another comment, a single `const projectReference = getAnnotationFromEntity(entity)` that hides the underlying implementation sounds better here